src/menu.cpp

```diff
@@ -16182,11 +16182,15 @@ void buttonGamemodsStartUploadItem(button_t* my)
 	{
 		if ( gamemods_window == 5 )
 		{
-			g_SteamWorkshop->SubmitItemUpdate("Item updated.");
+			char *tmp = strdup("Item updated.");
+			g_SteamWorkshop->SubmitItemUpdate(tmp);

```

Comment:
I believe this API can be changed to `const char*` instead, that should resolve the issue right?

Reply:
The API can be changed, yes, and I would also prefer that, but it currently makes no such promise. The bug would have to be reported upstream.

Reply:
You are correct. I did this patch in 2 hours before a doctor's appointment right after I got up, and assumed SubmitItemUpdate was Steam API itself, rather than your wrapper. Proper patch pushed now.